src/Input.js

```diff
@@ -5,7 +5,7 @@ import deprecationWarning from './utils/deprecationWarning';
 
 class Input extends InputBase {
   render() {
-    if (this.props.type === 'static') { // eslint-disable-line react/prop-types
+    if (this.props.type === 'static') {
```

Comment:
why remove this one? eslint doesn't pass travis for anything i've submitted that didn't have this, is there an issue or something that has been resolved?